### PR TITLE
fix: physical card status color AB#28503

### PR DESCRIPTION
**Pull request by arsforza**
[AB#28503](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/28503)

## Describe your changes

fixes:
- replaced ion-chip with badge
- updated colors used in card status
- applied wrap to card name to prevent status badge clipping in smaller screens

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: interfaces/Portal/src/app/components/registration-physical-card-overview/registration-physical-card-overview.component.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
-.card {
-  font-size: 14px;
+.pysical-card-number {
```
Comment (aberonni) [inline interfaces/Portal/src/app/components/registration-physical-card-overview/registration-physical-card-overview.component.scss:1]:
```suggestion
.physical-card-number {
```

File: interfaces/Portal/src/app/components/registration-physical-card-overview/registration-physical-card-overview.component.html [hunk 2]
```diff
@@ -22,21 +22,22 @@
           [detail-icon]="'card'"
           (click)="openCardDetails(card)"
         >
-          <ion-label class="card">
+          <ion-label
+            class="pysical-card-number util--half-margin-end"
```
Comment (aberonni) [inline interfaces/Portal/src/app/components/registration-physical-card-overview/registration-physical-card-overview.component.html:26]:
```suggestion
            class="physical-card-number util--half-margin-end"
```

**Review by aberonni — CHANGES_REQUESTED** [review]
Small typo but for the rest looks pretty straightforward ✌️

**Review by elwinschmitz — APPROVED** [review]
I think it would be even better to have a dedicated component for this (so we only have to do the "status-to-classname"-magic in 1 place;

But for now this is already an improvement. ;)

**Pull request merged**